components/badge/badge.html

```diff
@@ -0,0 +1,127 @@
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;900&display=swap" rel="stylesheet">
+    <!-- css file -->
+    <link href="../../styles.css" rel="stylesheet">
+    <title>badge</title>
+</head>
+<body>
+    <nav class="navigation-container">
+        <div class="nav-brand">FALCON</div>
+        <ul class="list-non-bullet nav-pills">
+            <li class="list-item">
+                <a class="link link-active" href="">HOME</a>
+            </li>
+            <li class="list-item">
+                <a class="link" href="">DOCS</a>
+            </li>
+            <li class="list-item">
+                <a class="link" href="">GITHUB</a>
+            </li>
+        </ul>
+    </nav>
+    <nav class="side-navigation-container">
+        <div class="nav header">
+            <h2>Getting Started</h2>
+            <div class="nav-content">
+                <a href="" >Introduction</a></div>
+            <div class="nav-content">
+                <a href="">Colors</a>
+            </div>
+            <div class="nav-content">
+                <a href="">Typography</a>
+            </div>
+
+        </div>
+        <div class="nav header">
+            <h2>Components</h2>
+            <div class="nav-content">
+                <a href="../alert/alert.html">Alerts</a></div>
+            <div class="nav-content">
+                <a href="../../components/avatar/avatar.html">Avatar</a>
+            </div>
+            <div class="nav-content">
+                <a href="../badge/badge.html">Badge</a>
+            </div>
+            <div class="nav-content">
+                <a href="../button/button.html">Button</a>
+            </div>
+            <div class="nav-content">
+                <a href="../card/card.html">Card</a>
+            </div>
+            <div class="nav-content">
+                <a href="../image/image.html">Image</a>
+            </div>
+            <div class="nav-content">
+                <a href="../input/input.html">Input</a>
+            </div>
+        </div>
+    </nav>
+
+    <!-- badge -->
+    <main class="page-content">
+        <section class="section-badge">
+            <h1>Badge</h1>
+            <p class="section-about">We see badges everywhere. Whether it be messages, notifications etc. </p>
+    
+            <h2 class="sub-header">Badge on avatar</h2>
+            <p class="section-about">Badge on avatar shows the current state of the user (online, offline or disabled) on any app . It has left and right positions.
+            <br>
+            There are four sizes of badges: xlg, lg, md and sm.</p>
+                <div class="badge-avatar">
+                <img class="badge-avatar-img-xlg" src="../../assets/images/avatar fox.jpg" />
+                <span class="badge badge-xlg-right"></span>
```

Comment:
Instead of naming it badge-avatar name it badge-container or badge-holder.
Maybe change span to div.